e2e/test/iothub/messaging/MessageReceiveE2EPoolAmqpTests.cs

```diff
@@ -160,45 +160,46 @@ await PoolingOverAmqp
             ConnectionStringAuthScope authScope = ConnectionStringAuthScope.Device)
         {
             // Initialize the service client
-            var serviceClient = ServiceClient.CreateFromConnectionString(TestConfiguration.IoTHub.ConnectionString);
-
-            async Task InitOperationAsync(IotHubDeviceClient deviceClient, TestDevice testDevice, TestDeviceCallbackHandler testDeviceCallbackHandler)
-            {
-                (Message msg, string payload, string p1Value) = MessageReceiveE2ETests.ComposeC2dTestMessage(Logger);
-
-                await testDeviceCallbackHandler.SetMessageReceiveCallbackHandlerAsync().ConfigureAwait(false);
-                testDeviceCallbackHandler.ExpectedMessageSentByService = msg;
-
-                await serviceClient.SendAsync(testDevice.Id, msg).ConfigureAwait(false);
-            }
-
-            async Task TestOperationAsync(IotHubDeviceClient deviceClient, TestDevice testDevice, TestDeviceCallbackHandler testDeviceCallbackHandler)
+            using (var serviceClient = new IotHubServiceClient(TestConfiguration.IoTHub.ConnectionString))
```

Comment:
Why switch to this style of using?

Reply:
oh, it just need a `using` keyword before `var` and you've added it. Just switch to the other style.